### PR TITLE
Cleanup: resolve build warnings

### PR DESCRIPTION
**Pull request by baum**
:broom: Cleanup: resolve build warnings

### CODE REVIEW

File: src/write_queue.rs [hunk 1]
```diff
@@ -76,20 +76,19 @@ impl WriteQueue {
...
+            .map_err(|_| GetObjectError::NoSuchKey(NoSuchKey::builder().build()))?;
         Ok(GetObjectOutput::builder().set_body(Some(stream)).build())
     }
 
     pub async fn head_object(
         &self,
-        mut i: HeadObjectInput,
+        _: HeadObjectInput,
     ) -> Result<HeadObjectOutput, HeadObjectError> {
-        // let fname = self.to_file_name(i.bucket(), i.key()) + ".s3d-object-md.yaml";
```
Comment (guymguym) [inline src/write_queue.rs:92]: Maybe you can leave that comment - it was a beginning idea for an impl

File: src/write_queue.rs [hunk 2]
```diff
@@ -76,20 +76,19 @@ impl WriteQueue {
...
     pub async fn get_object(
         &self,
-        mut i: GetObjectInput,
+        i: GetObjectInput,
     ) -> Result<GetObjectOutput, GetObjectError> {
         let fname = self.to_file_name(i.bucket(), i.key());
         let stream = ByteStream::from_path(Path::new(&fname))
             .await
-            .map_err(|err| GetObjectError::NoSuchKey(NoSuchKey::builder().build()))?;
+            .map_err(|_| GetObjectError::NoSuchKey(NoSuchKey::builder().build()))?;
```
Comment (guymguym) [inline src/write_queue.rs]: Another option is to leave the name but prefix the unused name with underscore, like `_err` which just helps to preserve the original meaning a bit more.

Comment (baum) [reply src/write_queue.rs]: Hope I've addressed your concerns....

**Review by guymguym — APPROVED** [review]
Thanks!

**Pull request merged**